### PR TITLE
1620 max file size exceeded handling

### PR DESCRIPTION
**Pull request by ledsoft**
https://kbss.felk.cvut.cz/redmine/issues/1620

### CODE REVIEW

File: src/test/java/cz/cvut/kbss/termit/service/config/ConfigurationProviderTest.java [hunk 1]
```diff
@@ -21,19 +19,16 @@
...
-    void setUp() {
-        this.sut = new ConfigurationProvider(config, service);
-    }
-
     @Test
     void getConfigurationReturnsConfigurationDtoWithRelevantConfigurationValues() {
         final ConfigurationDto result = sut.getConfiguration();
         assertNotNull(result);
         assertEquals(config.getPersistence().getLanguage(), result.getLanguage());
+        assertEquals(environment.getProperty("spring.servlet.multipart.max-file-size"), result.getMaxFileUploadSize());
```
Comment (psiotwo) [inline src/test/java/cz/cvut/kbss/termit/service/config/ConfigurationProviderTest.java]: Why not configure it from test/application.yml through Configuration

File: src/main/java/cz/cvut/kbss/termit/service/config/ConfigurationProvider.java [hunk 2]
```diff
@@ -21,9 +22,11 @@
 
     private final UserRoleRepositoryService service;
 
+    @Value("${spring.servlet.multipart.max-file-size}")
```
Comment (psiotwo) [inline src/main/java/cz/cvut/kbss/termit/service/config/ConfigurationProvider.java:25]: why is this not part of Configuration?

File: ontology/termit-glosář.ttl [hunk 3]
```diff
@@ -459,4 +459,13 @@ termit-pojem:má-můj-poslední-komentář
         <http://www.w3.org/2004/02/skos/core#inScheme>
                 termit:glosář ;
         <http://www.w3.org/2004/02/skos/core#prefLabel>
-                "Has my last comment"@en , "Má můj poslední komentář"@cs .
\ No newline at end of file
+                "Has my last comment"@en , "Má můj poslední komentář"@cs .
+
+termit-pojem:má-maximální-velikost-souboru
```
Comment (psiotwo) [inline ontology/termit-glosář.ttl:464]: why to have in the ontology?

File: src/main/java/cz/cvut/kbss/termit/service/config/ConfigurationProvider.java (hunk 2 above)
Comment (ledsoft) [reply src/main/java/cz/cvut/kbss/termit/service/config/ConfigurationProvider.java:25]: TermIt configuration contains only TermIt-related configuration, nothing Spring related.

File: src/test/java/cz/cvut/kbss/termit/service/config/ConfigurationProviderTest.java (hunk 1 above)
Comment (ledsoft) [reply src/test/java/cz/cvut/kbss/termit/service/config/ConfigurationProviderTest.java]: See below.

**Review by psiotwo — APPROVED** [review]
lgtm

**Pull request merged**